packages/nx/src/commands-runner/remove-projects-from-command-grpah.ts

```diff
@@ -0,0 +1,22 @@
+import { CommandGraph } from './command-graph';
+
+export function removeProjectsFromCommandGraph(
```

Comment:
Make the task graph utils typings more generic and use the same functions

Reply:
changed the original function